packages/inetutils.rb

```diff
@@ -40,5 +42,27 @@ def self.build
 
   def self.install
     system 'make', "DESTDIR=#{CREW_DEST_DIR}", 'install'
+    system "patchelf --set-rpath #{CREW_LIB_PREFIX} #{CREW_DEST_PREFIX}/bin/ping"
+    system "patchelf --set-rpath #{CREW_LIB_PREFIX} #{CREW_DEST_PREFIX}/bin/ping6"
+    system "patchelf --set-rpath #{CREW_LIB_PREFIX} #{CREW_DEST_PREFIX}/bin/traceroute"
+  end
+
+  def self.preinstall
+    system "sudo chown chronos #{CREW_PREFIX}/bin/ping" \
+      if File.exist? "#{CREW_PREFIX}/bin/ping"
+    system "sudo chown chronos #{CREW_PREFIX}/bin/ping6" \
+      if File.exist? "#{CREW_PREFIX}/bin/ping6"
+    system "sudo chown chronos #{CREW_PREFIX}/bin/traceroute" \
+      if File.exist? "#{CREW_PREFIX}/bin/traceroute"
```

Comment:
I can't think of any reason why the ownership would not be the chronos user unless these were installed using sudo which should never be the case.

Reply:
removed